examples/headers/bpf_helpers.h

```diff
@@ -8,6 +8,7 @@
  * in advance since bpf_helper_defs.h uses such types
  * as __u64.
  */
+#include "common.h"
```

Comment:
This will break the next time we sync changes from libbpf upstream.

Reply:
True, addressed in https://github.com/cilium/ebpf/pull/614.